Mention "serverless" for searches (#9835)
terraform-providers/terraform-provider-aws: website/docs/r/lambda_function.html.markdown

page_title: "AWS: aws_lambda_function"
sidebar_current: "docs-aws-resource-lambda-function"
description: |-

>   Provides a Lambda Function resource. Lambda allows you to trigger execution of code in response to events in AWS, also referred to as "serverless." The Lambda Function itself includes source code and runtime configuration.

Reviewer: ```suggestion
  Provides a Lambda Function resource. Lambda allows you to trigger execution of code in response to events in AWS, enabling serverless backend solutions. The Lambda Function itself includes source code and runtime configuration.
```

---

# Resource: aws_lambda_function

> Provides a Lambda Function resource. Lambda allows you to trigger execution of code in response to events in AWS, also referred to as _serverless_. The Lambda Function itself includes source code and runtime configuration.

Reviewer: ```suggestion
Provides a Lambda Function resource. Lambda allows you to trigger execution of code in response to events in AWS, enabling serverless backend solutions. The Lambda Function itself includes source code and runtime configuration.
```
